.github/workflows/broker.yml

```diff
@@ -92,7 +92,7 @@ jobs:
         uses: docker/build-push-action@v2.10.0
         with:
           context: .
-          file: ./Dockerfile
+          file: ./Dockerfile-broker
```

Comment:
Before merge I'd like experiment whether `Dockerfile` can be placed to respective `packages` directory. It didn't work some time ago, but might work now.

Reply:
I tried this but it didn't really work, but there could be a way